README.md

```diff
@@ -2,33 +2,18 @@
 
 [![GitHub Build Status](https://github.com/cisagov/cool-userservices-was-db/workflows/build/badge.svg)](https://github.com/cisagov/cool-userservices-was-db/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) [Terraform
-module](https://www.terraform.io/docs/modules/index.html) GitHub
-repository started.  This skeleton project contains [licensing
-information](LICENSE), as well as [pre-commit
-hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
+This is a Terraform deployment for creating the Web Application Scanning (WAS)
+database tables in the COOL User Services account.
```

Comment:
Minor nitpick but it might be nice to mention that they're tables in AWS DynamoDB here as well.

Reply:
Please behold the glory that is https://github.com/cisagov/cool-userservices-was-db/pull/11/commits/e6ef5784c969136b882e579e326465e24485019b.